README.md

```diff
@@ -49,6 +48,7 @@ Install the plugin with your favourite package manager:
 ```lua
 {
   "okuuva/auto-save.nvim",
+  version = '*', -- see https://devhints.io/semver, alternatively use '1' to avoid breaking changes
```

Comment:
I think this should by default point to latest major version. It's a chore to update it every time we release a new major version but that way new adopters will be guaranteed a stable experience.

Reply:
The problem is that we have configs for lazy, packer and plug in the readme. The `*` syntax is supported by all of them. Something like `version = '>=1.0.0'` seems to be only supported by lazy.
That's why I would leave it for now.

Reply:
Like you said, all of them support the wildcard syntax. So we should have them all pointing to the latest major version using that.